server/handler/frontend.go

```diff
@@ -60,14 +60,20 @@ func LoadTemplates(c *FilesConfig, basePath string, githubURL string) (templatet
 
 			return r
 		},
-		"hasRequires": func(requires common.Actors) bool {
-			return len(requires.Users) > 0 || len(requires.Teams) > 0 || len(requires.Organizations) > 0
+		"hasActors": func(requires *common.Requires) bool {
+			return len(requires.Actors.Users) > 0 || len(requires.Actors.Teams) > 0 || len(requires.Actors.Organizations) > 0
 		},
-		"getRequires": func(results *common.Result) map[string][]Membership {
-			return getRequires(results, strings.TrimSuffix(githubURL, "/"))
+		"hasPatterns": func(results *common.Result) bool {
+			return !results.Methods.IsEmpty()
```

Comment:
there's one thing I think I'm missing here and that's what if a github review approval (without comments) would satisfy the policy just fine? I think i can make it only show the Patterns if `github_review: true` and `github_review_comment_patterns` is not set.

Reply:
but then it's like the logic is slipping into the frontend a little bit.

Reply:
What if instead of thinking of this as patterns-only, we show all methods? I think that removes the need for `hasPatterns` (every rule will have methods) and then we can change `getPatterns` to `getMethods` and update the template accordingly (see later comments)

Reply:
sgtm